deployments/bsctestnet_addresses.json

```diff
@@ -52,7 +54,7 @@
     "VBNBAdmin": "0x04109575c1dbB4ac2e59e60c783800ea10441BBe",
     "VBNBAdmin_Implementation": "0x90c891bb2b1821ADE159ECa6FbA0418b2bD1b86D",
     "VBNBAdmin_Proxy": "0x04109575c1dbB4ac2e59e60c783800ea10441BBe",
-    "VBep20Delegate": "0xad6aa8Bb4829560412A94AA930745f407BF8000B",
```

Comment:
VBep20Delegate has been redeployed, I think this is not needed.

Reply:
I redeployed the mainnet version of vSolvBTC using the old implementation, and I kept the testnet deployment as is because the market is already added. I will make a testnet VIP changing the implementation to the existing one.